samples/metricscaler/metricscaler_test.py

```diff
@@ -109,32 +109,31 @@ def test_scale_bigtable(instance):
     instance.reload()
 
     cluster = instance.cluster(BIGTABLE_INSTANCE)
-    cluster.reload()
+
+    _nonzero_node_count = RetryInstanceState(
+        instance_predicate=lambda c: c.serve_nodes > 0,
```

Comment:
Due to the way that Bigtable assigns keys to be served by various nodes, I'm pretty sure that the minimum number of nodes per instance is 1, not 0.

https://github.com/googleapis/python-bigtable/blob/c9dafa922111eebd2efaa144403a6b1c48b6f9c1/samples/metricscaler/metricscaler.py#L92

Reply:
When initially creating an instance / cluster, the node count can be returned as 0:  see the [failure output](https://source.cloud.google.com/results/invocations/81bd1c0c-10b9-4539-86d6-f8447de02c80) for #159.